src/components/Sections/OrderSection/index.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+    <section class="order-section section">
+        <div class="container">
+            <div class="order-section__title-box">
+                <strong class="title">Check our sales for popular products</strong>
+                <a href="" class="order-section__sales-link link">All sales</a>
```

Comment:
if there is more than 1 attribute, it is customary to write it down one by one

`<a 
    href=""
    class="order-section__sales-link link"
 >
    All sales
</a>`

Reply:
ok, thanks